src/axolotl/common/cli.py

```diff
@@ -27,6 +27,8 @@ class TrainerCliArgs:
     merge_lora: bool = field(default=False)
     prepare_ds_only: bool = field(default=False)
     prompter: Optional[str] = field(default=None)
+    prompt_file: Optional[str] = field(default=None)
+    generation_cfg: Optional[str] = field(default=None)
```

Comment:
what should the value of generation_cfg look like? this doesn't seem obvious atm

Reply:
it should point to the config file "generation_config.json"

contents of the file could be:

{
  "bos_token_id": 1,
  "do_sample": true,
  "eos_token_id": 2,
  "pad_token_id": 0,
  "temperature": 0.6,  // 'randomness' of outputs, 0.0 is the min and 1.0 the max
  "top_p": 0.95,
  "repetition_penalty": 1.1,
  "max_new_tokens": 1024,  // max number of tokens to generate in the output
  "top_k": 40
}